build.gradle

```diff
@@ -99,6 +99,7 @@ dependencies {
 
     application name: "darcula", version: "1.0.0"
     application 'de.sciss:syntaxpane:1.1.+'
+	application 'me.xdrop:fuzzywuzzy:1.2.0'
```

Comment:
The indentation is pretty fuzzy-wuzzy. 😅

Reply:
haha, yeah. will fix.